### PR TITLE
Update --dockerfile option description to state docker-compose is supported

### PR DESCRIPTION
**Pull request by phil-d-wilson**
Change-type: patch
Signed-off-by: Phil Wilson <phil@balena.io>

### CODE REVIEW

File: lib/actions/push.ts [hunk 1]
```diff
@@ -187,7 +187,7 @@ export const push: CommandDefinition<
 			signature: 'dockerfile',
 			parameter: 'Dockerfile',
 			description:
-				'Alternative Dockerfile name/path, relative to the source folder',
+				'Alternative Dockerfile (or docker-compose file) name/path, relative to the source folder',
```
Comment (pdcastro) [inline lib/actions/push.ts:190]: Hmm.. Is that true though? :-)  I was under the impression that the CLI did not allow specifying an alternative `docker-compose` file. Did you test, did it work? Here's what I just got:
```
$ balena push 192.168.0.37 -s ~/balena/other/simple_multicontainer --dockerfile my-docker-compose.yml
[Info]    Starting build on device 192.168.0.37
[Info]    No "docker-compose.yml" file found at "/Users/paulo/balena/other/simple_multicontainer"
[Info]    Creating default composition with source: "/Users/paulo/balena/other/simple_multicontainer"
...
Some services failed to build:
	main: (HTTP code 400) unexpected - Dockerfile parse error line 1: unknown instruction: VERSION:
```

Comment (phil-d-wilson) [reply lib/actions/push.ts:190]: Yes, I did:

```
❯ balena push balena-dash --dockerfile .\docker-compose3.yml
Error: specified Dockerfile not found:
Specified dockerfile: "docker-compose3.yml"
Project's source folder: "."
Note that the specified Dockerfile path should be relative to the source folder.

ExpectedError: Error: specified Dockerfile not found:

⨯ phil@DESKTOP-7SPOFOE  ~\OneDrive\Documents\Source\balenaDash   BaseImageTest ≢ +3 ~1 -36 !                                                                                                        [13:33]
❯ balena push balena-dash --dockerfile .\docker-compose2.yml
- Packaging the project source...[Warn]    CRLF (Windows) line endings detected in file: C:\Users\phil\OneDrive\Documents\Source\balenaDash\.git\gitk.cache
- Uploading source package to balena cloud
```
I made a version 2 which worked, and tried specifying a file that did not exist (3).

Comment (phil-d-wilson) [reply lib/actions/push.ts:190]:
```
⨯ phil@DESKTOP-7SPOFOE  ~\OneDrive\Documents\Source\balenaDash   BaseImageTest ≢ +3 ~1 -36 !                                                                                                        [14:57]
❯ balena version
11.32.16
```

Comment (phil-d-wilson) [reply lib/actions/push.ts:190]: Ah....no it doesn't. I still had my original docker-compose.yml in place, and it was picking up that. It just ignores the option if it finds your compose. :-(

That's a real shame, as I just told a user it worked. :-(

Comment (pdcastro) [reply lib/actions/push.ts:190]: Even so, it's puzzling that it did complain about the contents of the Dockerfile -- perhaps the original docker-compose.yml file was pointing to a different Dockerfile, or not requiring one (`image` instruction).
There is a CLI issue for alternative `docker-compose` file names to be implemented: #1142

Comment (phil-d-wilson) [reply lib/actions/push.ts:190]: It definitely should complain.

**Pull request closed without merging**